eth/stagedsync/stage_headers.go

```diff
@@ -383,9 +381,7 @@ func startHandlingForkChoice(
 	if err != nil {
 		return nil, err
 	}
-	log.Info("MMDBG Forking point is", "blockNumber", forkingPoint)
-	isBedrockBlock := cfg.chainConfig.Optimism != nil && cfg.chainConfig.IsBedrock(header.Number.Uint64())
-	if forkingPoint < preProgress && (isBedrockBlock || cfg.chainConfig.Optimism == nil) {
+	if forkingPoint < preProgress {
```

Comment:
The condition for this 'if' has changed and no longer checks for bedrock/optimism